NeuralABC_tools/EigenGame_v1.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+
+def EigenGame2(data, n):
+    M = np.matmul(data.T, data)
+    dim = M.shape[0]
+    vecs = []
+    gradients = []
+    
+    step = 0.01
+    
+    for i in range(n):
```

Comment:
I think it is unnecessary to populate ```vecs``` with random floats as it is more efficient to define vecs as a zeros array with the shape of nXfeat without the need for a loop.